Extension/src/main.ts

```diff
@@ -163,7 +163,6 @@ function handleError(error: any): void {
     installationInformation.hasError = true;
     installationInformation.telemetryProperties['stage'] = installationInformation.stage;
     let errorMessage: string;
-    let channel: vscode.OutputChannel = util.getOutputChannel();
```

Comment:
> let channel:  [](start = 3, length = 14)

again, I don't understand why you want to call the function so many times.

Reply:
Making sure people are looking at my PR's? :) I threw the Logger together quickly and forgot about those quick hacky edits I did for it.